lib/datasource/remote/firebase/firebase_remote_config.dart

```diff
@@ -1,117 +1,88 @@
 const String _testnet_defaultV2EndpointUrl = "https://api-test.telosfoundation.io";
+// END - TESTNET CONFIG
 
 class _FirebaseRemoteConfigService {
-  final defaults = <String, dynamic>{
-    _featureFlagGuardianKey: false,
-    _activeEOSEndpointKey: _eosEndpoints,
-    _termsAndConditionsUrlKey: _termsAndConditionsDefaultUrl,
-    _privacyPolicyKey: _privacyPolicyUrl,
-    _hyphaEndPointKey: _hyphaEndPointUrl,
-    _explorerUrlKey: _explorerUrl,
-    _dhoExplorerUrlKey: _dhoExplorerUrl,
-    _defaultEndPointUrlKey: _defaultEndPointUrl,
-    _defaultV2EndPointUrlKey: _defaultV2EndpointUrl
-  };
+  late RemoteConfig _remoteConfig;
 
-  // RemoteConfig _remoteConfig;
   factory _FirebaseRemoteConfigService() => _instance;
 
   _FirebaseRemoteConfigService._();
 
   static final _FirebaseRemoteConfigService _instance = _FirebaseRemoteConfigService._();
 
+  final defaults = <String, dynamic>{
+    _activeEOSEndpointKey: _eosEndpoints,
+    _hyphaEndPointKey: _hyphaEndPointUrl,
+    _defaultEndPointUrlKey: _defaultEndPointUrl,
+    _defaultV2EndPointUrlKey: _defaultV2EndpointUrl
+  };
+
   void refresh() {
-    // Config has not been init yet. Dont call this function.
-    // if (_remoteConfig == null) {
-    //   return;
-    // }
-
-    // _remoteConfig.fetch().then((value) {
-    //   print(" _remoteConfig fetch worked");
-    //   _remoteConfig.activate().then((bool value) {
-    //     print(" _remoteConfig activate worked params were activated " + value.toString());
-    //   }).onError((error, stackTrace) {
-    //     print(" _remoteConfig activate failed");
-    //   });
-    // }).onError((error, stackTrace) {
-    //   print(" _remoteConfig fetch failed");
-    // });
+    _remoteConfig.fetch().then((value) {
+      print(" _remoteConfig fetch worked");
+      _remoteConfig.activate().then((bool value) {
+        print(" _remoteConfig activate worked params were activated $value");
+      }).onError((error, stackTrace) {
+        print(" _remoteConfig activate failed");
+      });
+    }).onError((error, stackTrace) {
+      print(" _remoteConfig fetch failed");
+    });
   }
 
   Future initialise() async {
-    // _remoteConfig = await RemoteConfig.instance;
-    //
-    // /// Maximum age of a cached config before it is considered stale. we set to 60 secs since we store important data.
-    // await _remoteConfig.setConfigSettings(RemoteConfigSettings(
-    //   minimumFetchInterval: const Duration(seconds: 60),
-    //   fetchTimeout: const Duration(seconds: 60),
-    // ));
-    //
-    // await _remoteConfig.setDefaults(defaults);
-    // refresh();
-  }
+    _remoteConfig = RemoteConfig.instance;
+    await _remoteConfig.setDefaults(defaults);
 
-  bool get featureFlagGuardiansEnabled => false; //_remoteConfig.getBool(_featureFlagGuardianKey);
+    /// Maximum age of a cached config before it is considered stale. we set to 60 secs since we store important data.
```

Comment:
does that mean it will be polling this every 60 seconds?

Reply:
no, it means that after 60 secs the values are marked as "staled" - and when a call to get values is executed, it will try to fetch the new values